ml-agents/mlagents/envs/brain.py

```diff
@@ -78,16 +91,16 @@ def from_agent_proto(agent_info_list, brain_params):
         brain_info = BrainInfo(
             visual_observation=vis_obs,
             vector_observation=np.nan_to_num(
-                np.array([x.stacked_vector_observation for x in agent_info_list])),
+                np.array([x.stacked_vector_observation for x in agent_info_list])).tolist(),
             text_observations=[x.text_observation for x in agent_info_list],
-            memory=memory,
+            memory=memory.tolist(),
```

Comment:
Just a thought: Is there a point in parsing the proto converting into an np.array and then converting into a list? As in can we cut out the numpy parsing?